driver-core/src/test/resources/change-streams/change-streams-errors.json

```diff
@@ -54,7 +54,9 @@
               "cursor": {},
               "pipeline": [
                 {
-                  "$changeStream": {}
+                  "$changeStream": {
```

Comment:
Turns out the tests were written incorrectly and we were ignoring the whole pipeline in the case where an error was expected.  Once I fixed that this test started failing, so I fixed it to match change-streams.json.  I was going to open a spec PR to make this change, but then i saw that this line had actually been removed from change-streams.json (argument being that it's the default, so drivers don't need to send it).  So once we sync the change stream tests, we'll have to fix the driver to not send the default value for fullDocument.